detection-rules/body_business_email_compromise_out_of_band_pivot.yml

```diff
@@ -0,0 +1,36 @@
+name: "Body: Business Email Compromise (BEC) with out of band pivot"
+description: |
+   Detects potential Business Email Compromise (BEC) attacks by analyzing text within the email body
+   looking specifically for language attempting to pivot the victim out of the protected email ecosystem as well as other suspicious indicators.
+type: "rule"
+severity: "medium"
+source: |
+  type.inbound
+  and length(body.current_thread.text) < 500
+  and regex.icontains(body.current_thread.text,
+                      "personal email|(whatsapp|cell(phone)?).{0,8}(#|number)"
+  )
+  and (
+    any(ml.nlu_classifier(body.current_thread.text).entities,
+        .name == "urgency" or .name == "request"
+    )
```

Comment:
do we need to exclude known senders for this bit of logic? this will basically flag on messages with "cell" and a request in it right?

Reply:
We were probably typing at the same time. Test rules were just kicked off. Right before you commented. It has to contain cell #|number and be short, testing looked good I'm curious how prone it will be without gating.